x-pack/plugins/endpoint/public/applications/endpoint/components/page_view.tsx

```diff
@@ -20,24 +20,65 @@ import React, { memo, ReactNode } from 'react';
 import styled from 'styled-components';
 
 const StyledEuiPage = styled(EuiPage)`
-  padding: 0;
+  &.endpoint--isListView {
+    padding: 0;
 
-  .endpoint-header {
-    padding: ${props => props.theme.eui.euiSizeL};
+    .endpoint-header {
+      padding: ${props => props.theme.eui.euiSizeL};
```

Comment:
i think we also want to do `margin: 0;` here. from the mockies i believe the header words are evensteven in the middle
![image](https://user-images.githubusercontent.com/56409205/78065424-467f6f00-7361-11ea-9939-f46753e8207a.png)

Reply:
GOOOD EYE 👏 
I will make this change in a subsequent PR. 

FYI:  The extra white space here is noticeable because a `bodyHeader` is also being used with the `PageView` component.